scripts/testpop/testpop.go

```diff
@@ -35,8 +35,13 @@ import (
 )
 
 var (
-	hostport        = flag.String("listen", "127.0.0.1:3000", "hostport to start ringpop on")
-	hostfile        = flag.String("hosts", "./hosts.json", "path to hosts file")
+	hostport = flag.String("listen", "127.0.0.1:3000", "hostport to start ringpop on")
+	hostfile = flag.String("hosts", "./hosts.json", "path to hosts file")
+
+	suspectPeriod   = flag.Int("suspect", 5, "suspect period in seconds")
```

Comment:
* Why not "--suspect-period="? They are called "long options", after all 😄  I think if it's just called "suspect" one might have to consult the --help to clarify what it means. (Consider "suspect=5" without context. What does it mean? "5 suspects"?)

* Why in seconds and not milliseconds? Can the value have a decimal place (e.g. 1.5)? Why not just use Go's built in notation (e.g user can specify "5s", or  "800ms")? I understand we'd probably want parity with main.js in Node too; so that's a consideration.